### ISSUE TITLE
Add new techniques to "How to Meet WCAG (Quick Reference)"

### ISSUE THREAD

**Issue by philljenkins** [w3c/wcag#3451]
This is probably on someone's to-do list, but the new SC and associated Techniques is in the docs yet
Such as 
https://www.w3.org/WAI/WCAG22/quickref/#focus-not-obscured-minimum
is missing

**Comment by yatil** [comment]
See the (subtle) note on the top: 
> WCAG 2.2 Note: Success Criteria and Techniques new in WCAG 2.2 are in the process of being added to this tool.

In the meantime, I have added the SCs to my local fork here: https://yatil.github.io/wai-wcag-quickref/

I’m currently cleaning up/adding the techniques as well.

**Comment by kfranqueiro** [comment]
This looks to be a stale issue, and has since been resolved.

(There was a period of time where Quickref's data fell out of sync, but that has also been resolved and a republish was performed today, so I'm now cleaning up related issues.)
